src/github.com/getlantern/flashlight/logging.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/dogenzaka/rotator"
+	"github.com/getlantern/appdir"
+	"github.com/getlantern/golog"
+	"github.com/getlantern/jibber_jabber"
+	"github.com/getlantern/wfilter"
+	"github.com/segmentio/go-loggly"
+)
+
+var (
+	LogTimestampFormat = "Jan 02 15:04:05.000"
+	log                = golog.LoggerFor("flashlight")
+	logglyToken        string
+	versionToLoggly    = fmt.Sprintf("%v (%v)", version, buildDate)
+	lang               string
+	country            string
+	tz                 string
+)
+
+func init() {
+	lang, _ = jibber_jabber.DetectLanguage()
+	country, _ = jibber_jabber.DetectTerritory()
```

Comment:
I think we should use the geoip information from @xiam instead of this.  DetectTerritory is looking at language settings, whereas what we want is actual physical location.